<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sonata_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>자유 렌트카</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 81 14pt &quot;나눔스퀘어라운드 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>쏘나타 뉴라이즈</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>중형</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
      </property>
      <property name="text">
       <string>         LPG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 81 12pt &quot;나눔스퀘어라운드 ExtraBold&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>1일 24,000원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="genesis_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>공항 렌트카</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 81 14pt &quot;나눔스퀘어라운드 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>제네시스 EQ900</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>고급</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
      </property>
      <property name="text">
       <string>      휘발유</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 81 12pt &quot;나눔스퀘어라운드 ExtraBold&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>1일 90,000원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>제주공항 렌트카</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 81 14pt &quot;나눔스퀘어라운드 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>더 뉴 그랜드</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
        </property>
        <property name="text">
         <string>SUV/승합</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;나눔스퀘어라운드 Regular&quot;;</string>
      </property>
      <property name="text">
       <string>             LPG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">font: 81 12pt &quot;나눔스퀘어라운드 ExtraBold&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>   1일 40,000원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="thenewgrand_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>620</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>렌터카 서비스 포함 취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
